### DISCUSSION TITLE
Font Awesome 6

### DISCUSSION THREAD

**Discussion by MichaelBeever** [skartknet/Iconic#38] [General]
Hi,

Has anyone got Font Awesome pro 6 icons working? I am trying to get the  webfonts but can't find them :(

Thanks

**Comment by skartknet** [comment]
I don't think Iconic will be able to use FA 6. The new icons package seems to just use a javascript file to load everything.

**Comment by thenexus00** [comment]
Pre configured is in there for them but it defiantly does not work.
